certs_wallet/src/chaincert.rs

```diff
@@ -0,0 +1,22 @@
+//! Module Chaincert
+//!
+//! Module responsible of managing `Chaincerts` information and defining its corresponding struct.
+use crate::option::OptU64;
+use soroban_sdk::{contracttype, Address, Bytes};
+
+#[derive(Clone, PartialEq, Eq, Debug)]
+#[contracttype]
+/// The `Chaincert` information stored in the wallet
+pub struct Chaincert {
+    pub cid: Bytes,
+    /// Address of the governance contract that distributed the `Chaincert`
+    pub cont_dist: Address,
+    /// The id of the organization that distributed the `Chaincert`
+    pub org_id: Bytes,
+    /// The distribution date in Unix Timestamp format
+    pub dist_date: u64,
+    /// The expiration date in Unix Timestamp format
+    pub exp_date: OptU64,
+    /// A logical indicator that lets know if a `Chaincert` is revoked or not
+    pub revoked: bool,
+}
```

Comment:
Do we need an implementation here for creating instances of this struct?

Reply:
@L-Zuluaga Since the implementation isn't needed in any part of the code for the moment I didn't  do it(so I could avoid doing a non-relevant test), anyways I can create it if it is strictly necessary for this issue

Reply:
OK that makes sense, thanks for the clarification 🙏🏻 
In that case I don't think is strictly required for the issue, but I saw that you already implemented it 😅 so lets proceed merging 🚀